cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc

```diff
@@ -0,0 +1,27 @@
+[id=creating-instances-of-services-managed-by-the-operators]
+= Creating instances of services managed by the Operators
+include::modules/developer-cli-odo-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: creating-instances-of-services-managed-by-the-operators
+toc::[]
+
+:FeatureName: Creating instances of services managed by the Operators  in {odo-title}
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Operators are a method of packaging, deploying, and managing Kubernetes services. With odo, you can create instances of services from the Custom Resource Definitions (CRDs) provided by the Operators. You can then use these instances in your projects and link them to your components..
+
+To create services from an Operator, you must ensure that the Operator has valid values defined in its `metadata` to start the requested service. odo uses the `metadata.annotations.alm-examples` YAMl file of an Operator to start
+the service. If this YAML has placeholder values or sample values, a service cannot start. You can modify the YAML file and start the service with the modified values. See xref:Creating services from a YAML file
+
+.Prerequisites
+    • `oc` is installed
```

Comment:
Formatting for bullets is: 
"* Install the `oc` CLI."

s/• `oc` is installed/* Install the `oc` CLI and log in. 

For the following, maybe make it a sub-bullet (as follows), and for the "see Adding Operators to the cluster" part, you can link to another assembly. 
** Note that the configuration of the cluster determines the services available to you. To access Operator services, a cluster administrator must install the respective Operator on the cluster first. To learn more, see Adding Operators to the cluster. 

s/• Latest version of `odo` is installed/* Install the `odo` CLI. 
s/• Experimental mode is enabled./* Enable experimental mode. 
** To enable experimental mode in `odo` preferences, run:
<add code block, and format the code example: ---->
`odo preference set Experimental true` or use the environment variable `odo config set --env ODO_EXPERIMENTAL=true`
You can also use the `ODO_EXPERIMENTAL=true` prefix when running commands, for example `ODO_EXPERIMENTAL=true odo catalog list services`